phoenicis-scripts/src/main/java/org/phoenicis/scripts/wizard/UiSetupWizardImplementation.java

```diff
@@ -256,7 +257,8 @@ public MenuItem menu(String textToShow, List<String> menuItems) {
      */
     @Override
     public MenuItem menu(String textToShow, List<String> menuItems, String defaultValue) {
-        return messageSender.runAndWait(message -> setupUi.showMenuStep(message, textToShow, menuItems, defaultValue));
+        final List<String> copiedMenuItems = new ArrayList<>(menuItems);
```

Comment:
I think we should replace our `new ArrayList<>(...)` constructs with `List.copyOf(...)` especially when the list is immutable.

Reply:
Done

Reply:
That did not change anything.

Reply:
Then the `import java.util.ArrayList;` is not required anymore.

Reply:
@Zemogiter I have retried and it works perfectly. Are you sure that you are running the right code?

Reply:
I'm positive. I've copied the changes from the latest commit in this PR to the local source and compiled it then created a deb file and installed it.

Reply:
with mvn clean install?

Reply:
Can you try mvn exec:java inside phoencis-javafx directory?

Reply:
I was using mvn clean install as usual. And when used mvn exec:java I got this:
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.google.inject.internal.cglib.core.$ReflectUtils$1 (file:/usr/share/maven/lib/guice.jar) to method java.lang.ClassLoader.defineClass(java.lang.String,byte[],int,int,java.security.ProtectionDomain)
WARNING: Please consider reporting this to the maintainers of com.google.inject.internal.cglib.core.$ReflectUtils$1
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release
[INFO] Scanning for projects...
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/versions-maven-plugin/2.7/versions-maven-plugin-2.7.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/versions-maven-plugin/2.7/versions-maven-plugin-2.7.pom (17 kB at 23 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/versions-maven-plugin/2.7/versions-maven-plugin-2.7.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/versions-maven-plugin/2.7/versions-maven-plugin-2.7.jar (310 kB at 307 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/exec-maven-plugin/1.6.0/exec-maven-plugin-1.6.0.pom
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/exec-maven-plugin/1.6.0/exec-maven-plugin-1.6.0.pom (13 kB at 81 kB/s)
Downloading from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/exec-maven-plugin/1.6.0/exec-maven-plugin-1.6.0.jar
Downloaded from central: https://repo.maven.apache.org/maven2/org/codehaus/mojo/exec-maven-plugin/1.6.0/exec-maven-plugin-1.6.0.jar (58 kB at 326 kB/s)
[INFO] 
[INFO] -------------------< org.phoenicis:phoenicis-javafx >-------------------
[INFO] Building phoenicis-javafx 5.0-SNAPSHOT
[INFO] --------------------------------[ jar ]---------------------------------
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-configuration/5.0-SNAPSHOT/maven-metadata.xml
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-configuration/5.0-SNAPSHOT/phoenicis-configuration-5.0-SNAPSHOT.pom
[WARNING] The POM for org.phoenicis:phoenicis-configuration:jar:5.0-SNAPSHOT is missing, no dependency information available
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-repository/5.0-SNAPSHOT/maven-metadata.xml
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-repository/5.0-SNAPSHOT/phoenicis-repository-5.0-SNAPSHOT.pom
[WARNING] The POM for org.phoenicis:phoenicis-repository:jar:5.0-SNAPSHOT is missing, no dependency information available
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-scripts/5.0-SNAPSHOT/maven-metadata.xml
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-scripts/5.0-SNAPSHOT/phoenicis-scripts-5.0-SNAPSHOT.pom
[WARNING] The POM for org.phoenicis:phoenicis-scripts:jar:5.0-SNAPSHOT is missing, no dependency information available
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-library/5.0-SNAPSHOT/maven-metadata.xml
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-library/5.0-SNAPSHOT/phoenicis-library-5.0-SNAPSHOT.pom
[WARNING] The POM for org.phoenicis:phoenicis-library:jar:5.0-SNAPSHOT is missing, no dependency information available
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-engines/5.0-SNAPSHOT/maven-metadata.xml
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-engines/5.0-SNAPSHOT/phoenicis-engines-5.0-SNAPSHOT.pom
[WARNING] The POM for org.phoenicis:phoenicis-engines:jar:5.0-SNAPSHOT is missing, no dependency information available
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-containers/5.0-SNAPSHOT/maven-metadata.xml
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-containers/5.0-SNAPSHOT/phoenicis-containers-5.0-SNAPSHOT.pom
[WARNING] The POM for org.phoenicis:phoenicis-containers:jar:5.0-SNAPSHOT is missing, no dependency information available
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-settings/5.0-SNAPSHOT/maven-metadata.xml
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-settings/5.0-SNAPSHOT/phoenicis-settings-5.0-SNAPSHOT.pom
[WARNING] The POM for org.phoenicis:phoenicis-settings:jar:5.0-SNAPSHOT is missing, no dependency information available
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-configuration/5.0-SNAPSHOT/phoenicis-configuration-5.0-SNAPSHOT.jar
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-library/5.0-SNAPSHOT/phoenicis-library-5.0-SNAPSHOT.jar
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-scripts/5.0-SNAPSHOT/phoenicis-scripts-5.0-SNAPSHOT.jar
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-repository/5.0-SNAPSHOT/phoenicis-repository-5.0-SNAPSHOT.jar
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-engines/5.0-SNAPSHOT/phoenicis-engines-5.0-SNAPSHOT.jar
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-containers/5.0-SNAPSHOT/phoenicis-containers-5.0-SNAPSHOT.jar
Downloading from jitpack.io: https://jitpack.io/org/phoenicis/phoenicis-settings/5.0-SNAPSHOT/phoenicis-settings-5.0-SNAPSHOT.jar
[INFO] ------------------------------------------------------------------------
[INFO] BUILD FAILURE
[INFO] ------------------------------------------------------------------------
[INFO] Total time:  8.687 s
[INFO] Finished at: 2019-06-11T21:18:45+02:00
[INFO] ------------------------------------------------------------------------
[ERROR] Failed to execute goal on project phoenicis-javafx: Could not resolve dependencies for project org.phoenicis:phoenicis-javafx:jar:5.0-SNAPSHOT: The following artifacts could not be resolved: org.phoenicis:phoenicis-configuration:jar:5.0-SNAPSHOT, org.phoenicis:phoenicis-repository:jar:5.0-SNAPSHOT, org.phoenicis:phoenicis-scripts:jar:5.0-SNAPSHOT, org.phoenicis:phoenicis-library:jar:5.0-SNAPSHOT, org.phoenicis:phoenicis-engines:jar:5.0-SNAPSHOT, org.phoenicis:phoenicis-containers:jar:5.0-SNAPSHOT, org.phoenicis:phoenicis-settings:jar:5.0-SNAPSHOT: Could not find artifact org.phoenicis:phoenicis-configuration:jar:5.0-SNAPSHOT in jitpack.io (https://jitpack.io) -> [Help 1]
[ERROR] 
[ERROR] To see the full stack trace of the errors, re-run Maven with the -e switch.
[ERROR] Re-run Maven using the -X switch to enable full debug logging.
[ERROR] 
[ERROR] For more information about the errors and possible solutions, please read the following articles:
[ERROR] [Help 1] http://cwiki.apache.org/confluence/display/MAVEN/DependencyResolutionException
```

Reply:
Maven is trying to download phoenicis. I suspect that it is not properly installed inside .m2